src/css/components/main-content.scss

```diff
@@ -3,8 +3,8 @@
 @import '../override_vars.scss';
 @import '../vars.scss';
 
-.content {
-  background: $color-white;
+.content.content-text {
+  background-color: $color-white;
```

Comment:
These padding and background color values should apply only to the docs. This targets the docs.